README.md

```diff
@@ -6,73 +6,147 @@ Welcome to the monorepo of the Liferay Frontend Infrastructure team.
-| [`js-themes-toolkit` v8](./maintenance/projects/js-themes-toolkit-v8-x) | [label: `js-themes-toolkit`, `8.x`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Ajs-themes-toolkit+label%3A8.x)   | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=js-themes-toolkit,8.x)  |
+### Documentation
 
-### `js-toolkit`
+These are the documents that used to live in the [liferay-frontend-guidelines](https://github.com/liferay/liferay-frontend-guidelines) repository.
 
-| Project                                              | Issues & PRs                                                                                                                                    | Actions                                                                                            |
-| ---------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
-| [`js-toolkit` v3](./projects/js-toolkit)             | [label: `js-toolkit`, `3.x`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Ajs-toolkit+label%3A3.x) | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=js-toolkit,3.x) |
-| [`js-toolkit` v2](./maintenance/projects/js-toolkit) | [label: `js-toolkit`, `2.x`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Ajs-toolkit+label%3A2.x) | [New issue](https://github.com/liferay/liferay-frontend-projects/issues/new?labels=js-toolkit,2.x) |
+-   Projects
+    -   [`guidelines/`](guidelines): High-level introduction to the guidelines.
+    -   [`guidelines/general/`](guidelines/general): General guidance about working in any Liferay codebase.
+    -   [`guidelines/dxp/`](guidelines/dxp): Guidance specific to working on Liferay DXP.
+    -   [`guidelines/css/`](guidelines/css): Guidance about CSS.
+-   Issues
+    -   [label: `guidelines`, `dependencies`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Aguidelines+label%3Adependencies): Requests to add new development dependencies
+    -   [label: `guidelines`, `rfc`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Aguidelines+label%3Arfc): Proposals for changes to guidelines
+    -   [label: `guidelines`, `question`](https://github.com/liferay/liferay-frontend-projects/issues?q=is%3Aissue+is%3Aopen+label%3Aguidelines+label%3Aquestion): Question about Liferay's frontend practices
+-   Pull requests
+    -   [label: `quidelines`](https://github.com/liferay/liferay-frontend-projects/pulls?q=is%3Apr+label%3Aguidelines+is%3Aopen)
```

Comment:
quidelines → guidelines

Reply:
Maybe we can rename everything else to `quideline` and leave this commit as is :thinking: